.github/workflows/develop.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
         strategy:
             matrix:
-                python-version: [3.7, 3.8]
```

Comment:
I think adding Python 3.6 here will break the nightly build again, since `aiida-core` no longer supports Python v3.6 (due to the pymatgen update of its numpy dependency).

Reply:
Good one, removed it again from the develop workflow